core/db/migrate/20141217215630_update_product_slug_index.rb

```diff
@@ -0,0 +1,6 @@
+class UpdateProductSlugIndex < ActiveRecord::Migration
+  def change
```

Comment:
with tables with many items, we've had performance issues from locking up the table to add the indexes. would you be opposed to adding `disable_ddl_transaction!` ?